utopia/components.utopia.js

```diff
@@ -356,7 +356,7 @@ const Components = {
       },
     },
     SpecialRow: {
```

Comment:
this needs to be renamed too otherwise it will throw an error in the validation. this name must match the name of the `component: ProductFeatureRow,` entry

Reply:
Fixed!